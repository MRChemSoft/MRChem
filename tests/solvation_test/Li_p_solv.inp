# vim:syntax=sh:

rel_prec = 1.0e-4               # Overall relative precision

MPI {
  numerically_exact = true
  share_coulomb_density = true
  share_coulomb_potential = true
  share_nuclear_potential = true
  share_xc_density = false
  share_xc_potential = false
}

MRA {
  min_scale = -5                  # Root scale defines world size 2^{-n}
  corner = [ -1, -1, -1 ]         # Translation of corner root box [x,y,z]
  boxes = [ 2, 2, 2 ]             # Number of root boxes [x,y,z]
}

Molecule {
  angstrom = false
  multiplicity = 1
  charge = 1
$coords
Li      0.0000  0.0000   0.0000
$end
}

WaveFunction {
  method = HF                  # Wave function method (HF or DFT)
  restricted = true              # Unrestricted calculation
}

Properties {
  scf_energy = true               # Compute ground state energy
  solvent_effect = true
}

Solvent {
  epsilon = 2
  atom_based_cavity = false
  $cavity
  1   0.0000   0.0000   0.0000   4.0000
  $end
  sigma = 0.2
  linear = false
}



SCF {
  run = true                     # Use initial guess, don't optimize
  initial_guess = SAD_DZ             # Reading precomputed GTO MOs
}
