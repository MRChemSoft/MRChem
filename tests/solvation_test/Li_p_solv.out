


************************************************************
***     __  __ ____   ____ _                             ***
***    |  \/  |  _ \ / ___| |__   ___ _ __ ___           ***
***    | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \          ***
***    | |  | |  _ <| |___| | | |  __/ | | | | |         ***
***    |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|         ***
***                                                      ***
***    VERSION 0.0.1 (rev. ac175b4)                      ***
***                                                      ***
***    Stig Rune Jensen <stig.r.jensen@uit.no>           ***
***    Luca Frediani    <luca.frediani@uit.no>           ***
***    Peter Wind       <peter.wind@uit.no>              ***
***                                                      ***
************************************************************


+++ Parallel execution: 
  MPI hosts available     : 1
  Threads/host            : 4
  Total used CPUs         : 4


------------------------------------------------------------

 MRCPP version   : X.Y.Z
 Git revision    : 0d899f8

 Linear algebra  : EIGEN
 Parallelization : OpenMP

------------------------------------------------------------




============================================================
                  MultiResolution Analysis                  
------------------------------------------------------------
 polynomial order = 8
 polynomial type  = Interpolating
------------------------------------------------------------
 total boxes      = 8
 boxes            = [           2           2           2 ]
 unit lengths     = [    32.00000    32.00000    32.00000 ]
 scaling factor   = [     1.00000     1.00000     1.00000 ]
 lower bounds     = [   -32.00000   -32.00000   -32.00000 ]
 upper bounds     = [    32.00000    32.00000    32.00000 ]
 total length     = [    64.00000    64.00000    64.00000 ]
============================================================


i run a string
============================================================
                          Molecule
------------------------------------------------------------
 Nr  Element             x             y             z      
------------------------------------------------------------
  1     Li          0.00000e+00   0.00000e+00   0.00000e+00
------------------------------------------------------------
 Center of mass:    0.00000e+00   0.00000e+00   0.00000e+00
============================================================


============================================================
                Setting up nuclear potential
------------------------------------------------------------
 Nr  Element         Charge        Precision     Smoothing 
------------------------------------------------------------
  1     Li           3.00000e+00   1.00000e-05   5.63587e-04
------------------------------------------------------------
 Projecting potential                 1032      3.21415e-01
============================================================


============================================================
                Setting up nuclear potential
------------------------------------------------------------
 Nr  Element         Charge        Precision     Smoothing 
------------------------------------------------------------
  1     Li           3.00000e+00   1.00000e-05   5.63587e-04
------------------------------------------------------------
 Projecting potential                 1032      3.40235e-01
============================================================


============================================================
              Projecting Gaussian-type density
------------------------------------------------------------
 Nr  Element                                 Rho_i
------------------------------------------------------------
  0     Li                            2.999999999995785e+00
------------------------------------------------------------
                 Wall time: 2.47333e+01 sec 
============================================================


============================================================
                  Projecting Hydrogen AOs
------------------------------------------------------------
    N    Atom   Label                     SquareNorm
------------------------------------------------------------
    1    Li1     1s                   1.000000000117468e+00
    2    Li1     2s                   1.000000000236291e+00
    3    Li1     2p                   9.999999999984690e-01
    4    Li1     2p                   9.999999999984689e-01
    5    Li1     2p                   9.999999999984689e-01
    6    Li1     3s                   1.000000000069046e+00
    7    Li1     3p                   9.999999999950161e-01
    8    Li1     3p                   9.999999999950161e-01
    9    Li1     3p                   9.999999999950161e-01
------------------------------------------------------------
                 Wall time: 1.03105e+00 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Coulomb potential                     328      4.18165e+00
 XC evaluate xcfun                     912      1.42345e+00
 XC energy                             456      1.13837e-04
 XC potential                          456      1.35842e-02
------------------------------------------------------------
                 Wall time: 5.62070e+00 sec 
============================================================


============================================================
                  Diagonalize Fock matrix
------------------------------------------------------------
 Compute Fock matrix                            6.62359e+00
 Diagonalize Fock matrix                        3.77190e-05
 Rotate orbitals                                1.08787e-01
------------------------------------------------------------
                 Wall time: 6.73396e+00 sec 
============================================================


============================================================
 OrbitalVector:   1 orbitals     1 occupied     2 electrons
------------------------------------------------------------
   n  RankID           Norm          Spin Occ      Error    
------------------------------------------------------------
   0     0       1.000000000000e+00    p   2    1.00000e+00
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-05
------------------------------------------------------------
 Coulomb density                       584      1.79348e-01
 Coulomb potential                     328      4.58151e+00
4.000000000000000e+00
1.485119372764015e+01 1
1
5.142709252369540e+00 2
2
1.186904197431564e+00 3
3
2.050811012895863e-01 4
4
2.817275285984191e-02 5
5
3.172959234112656e-03 6
6
2.954585576489190e-04 7
7
2.234493060083668e-05 8
8
1.278534368011467e-06 9
9
converged V_n
 Hartree-Fock exchange                 584      3.89224e+00
------------------------------------------------------------
                 Wall time: 1.66474e+02 sec 
============================================================


============================================================
                  Calculating Fock matrix
------------------------------------------------------------
 Kinetic part                         3.341543150000000e-01
 Potential part                       5.295287130000000e-01
------------------------------------------------------------
                 Wall time: 8.63835e-01 sec 
============================================================




####################### SCF cycle  1 #######################


============================================================
 Current precision                              1.00000e-05
------------------------------------------------------------
 Orbital threshold                              1.00000e-03
 Property threshold                             1.00000e-06
============================================================


============================================================
                  Digonalizing Fock matrix
------------------------------------------------------------
 Computing Lowdin matrix                        3.12830e-03
 Diagonalizing matrix                           4.01700e-06
 Rotating orbitals                              6.33954e-02
------------------------------------------------------------
                 Wall time: 6.66773e-02 sec 
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
 Nuclear energy                      -1.891129356755301e-01
 Electronic energy                   -7.064302919756155e+00
------------------------------------------------------------
 Total energy                        -7.253415855431685e+00
------------------------------------------------------------
                 Wall time: 9.50069e-01 sec 
============================================================


============================================================
                Applying Helmholtz operators
------------------------------------------------------------
 Orb    RealNorm   Nodes     ImagNorm   Nodes     Timing
------------------------------------------------------------
  0    1.01278e+00   520    0.00000e+00     0   1.01133e+01
------------------------------------------------------------
                 Wall time: 1.01135e+01 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-05
------------------------------------------------------------
 Coulomb density                       584      1.84839e-01
 Coulomb potential                     328      3.90252e+00
4.000000000000000e+00
5.032793669895674e-02 1
1
2.294606939863610e-02 2
2
7.950322270275874e-03 3
3
1.835070840025513e-03 4
4
3.170908293599820e-04 5
5
4.356376850382951e-05 6
6
4.906522471413473e-06 7
7
converged V_n
 Hartree-Fock exchange                 520      3.60857e+00
------------------------------------------------------------
                 Wall time: 1.10270e+02 sec 
============================================================


============================================================
                  Calculating Fock matrix
------------------------------------------------------------
 Kinetic part                         3.179988430000000e-01
 Potential part                       5.063226480000000e-01
------------------------------------------------------------
                 Wall time: 8.25744e-01 sec 
============================================================


============================================================
                          Orbitals
------------------------------------------------------------
 Orb    F(i,i)        Error         nNodes  Spin  Occ  Done 
------------------------------------------------------------
  0  -2.66078e+00   6.00009e-02        520    p    2    0
------------------------------------------------------------
 Total error:                            6.00009e-02    0
============================================================


============================================================
                    Energy                 Update      Done 
------------------------------------------------------------
 Orbital      -5.271548751219200e+00    -5.27155e+00    0
 Kinetic       7.926568097038396e+00     7.92657e+00    0
 N-E          -1.653151433181665e+01    -1.65315e+01    0
 Coulomb       3.333397281439600e+00     3.33340e+00    0
 Exchange     -1.666698539660074e+00    -1.66670e+00    0
 X-C           0.000000000000000e+00     0.00000e+00    1
------------------------------------------------------------
 Electronic   -7.064302919756155e+00    -7.06430e+00    0
 Nuclear      -1.891129356755301e-01    -1.89113e-01    0
------------------------------------------------------------
 Total        -7.253415855431685e+00    -7.25342e+00    0
============================================================


################ Wall time: 1.22456e+02 sec ################




####################### SCF cycle  2 #######################


============================================================
 Current precision                              1.00000e-05
------------------------------------------------------------
 Orbital threshold                              1.00000e-03
 Property threshold                             1.00000e-06
============================================================


============================================================
                  Digonalizing Fock matrix
------------------------------------------------------------
 Computing Lowdin matrix                        2.51147e-03
 Diagonalizing matrix                           3.52300e-06
 Rotating orbitals                              5.86587e-02
------------------------------------------------------------
                 Wall time: 6.12790e-02 sec 
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
 Nuclear energy                      -1.890211471831235e-01
 Electronic energy                   -7.107575563408417e+00
------------------------------------------------------------
 Total energy                        -7.296596710591541e+00
------------------------------------------------------------
                 Wall time: 8.86246e-01 sec 
============================================================


============================================================
                Applying Helmholtz operators
------------------------------------------------------------
 Orb    RealNorm   Nodes     ImagNorm   Nodes     Timing
------------------------------------------------------------
  0    1.00365e+00   520    0.00000e+00     0   8.32230e+00
------------------------------------------------------------
                 Wall time: 8.32280e+00 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-05
------------------------------------------------------------
 Coulomb density                       584      1.38523e-01
 Coulomb potential                     328      3.39567e+00
4.000000000000000e+00
9.136866101714486e-03 1
1
1.481019757905375e-04 2
2
5.244202747522156e-05 3
3
1.234893745288333e-05 4
4
2.171287411763701e-06 5
5
converged V_n
 Hartree-Fock exchange                 520      3.55592e+00
------------------------------------------------------------
                 Wall time: 8.13126e+01 sec 
============================================================


============================================================
                  Calculating Fock matrix
------------------------------------------------------------
 Kinetic part                         3.213478160000000e-01
 Potential part                       5.260024490000000e-01
------------------------------------------------------------
                 Wall time: 8.47493e-01 sec 
============================================================


============================================================
                          Orbitals
------------------------------------------------------------
 Orb    F(i,i)        Error         nNodes  Spin  Occ  Done 
------------------------------------------------------------
  0  -2.66424e+00   1.26234e-02        520    p    2    0
------------------------------------------------------------
 Total error:                            1.26234e-02    0
============================================================


============================================================
                    Energy                 Update      Done 
------------------------------------------------------------
 Orbital      -5.321559497128622e+00    -5.00107e-02    0
 Kinetic       7.636386073961290e+00    -2.90182e-01    0
 N-E          -1.627792829009147e+01     2.53586e-01    0
 Coulomb       3.319982486948035e+00    -1.34148e-02    0
 Exchange     -1.659991127447254e+00     6.70741e-03    0
 X-C           0.000000000000000e+00     0.00000e+00    1
------------------------------------------------------------
 Electronic   -7.107575563408417e+00    -4.32726e-02    0
 Nuclear      -1.890211471831235e-01     9.17885e-05    0
------------------------------------------------------------
 Total        -7.296596710591541e+00    -4.31809e-02    0
============================================================


################ Wall time: 9.15842e+01 sec ################




####################### SCF cycle  3 #######################


============================================================
 Current precision                              1.00000e-05
------------------------------------------------------------
 Orbital threshold                              1.00000e-03
 Property threshold                             1.00000e-06
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
 Nuclear energy                      -1.890205386758395e-01
 Electronic energy                   -7.110131319967708e+00
------------------------------------------------------------
 Total energy                        -7.299151858643548e+00
------------------------------------------------------------
                 Wall time: 9.33002e-01 sec 
============================================================


============================================================
                Applying Helmholtz operators
------------------------------------------------------------
 Orb    RealNorm   Nodes     ImagNorm   Nodes     Timing
------------------------------------------------------------
  0    1.00138e+00   520    0.00000e+00     0   8.23017e+00
------------------------------------------------------------
                 Wall time: 8.23103e+00 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-05
------------------------------------------------------------
 Coulomb density                       584      1.42606e-01
 Coulomb potential                     328      3.35276e+00
4.000000000000000e+00
2.935543713465611e-03 1
1
4.302176041685047e-06 2
2
converged V_n
 Hartree-Fock exchange                 520      3.48471e+00
------------------------------------------------------------
                 Wall time: 4.21508e+01 sec 
============================================================


============================================================
                  Calculating Fock matrix
------------------------------------------------------------
 Kinetic part                         3.321394050000000e-01
 Potential part                       5.049082900000000e-01
------------------------------------------------------------
                 Wall time: 8.37201e-01 sec 
============================================================


============================================================
                          Orbitals
------------------------------------------------------------
 Orb    F(i,i)        Error         nNodes  Spin  Occ  Done 
------------------------------------------------------------
  0  -2.66544e+00   3.03136e-03        520    p    2    0
------------------------------------------------------------
 Total error:                            3.03136e-03    0
============================================================


============================================================
                    Energy                 Update      Done 
------------------------------------------------------------
 Orbital      -5.328472551441882e+00    -6.91305e-03    0
 Kinetic       7.546219269212614e+00    -9.01668e-02    0
 N-E          -1.618596033262756e+01     9.19680e-02    0
 Coulomb       3.311268357254940e+00    -8.71413e-03    0
 Exchange     -1.655634101268410e+00     4.35703e-03    0
 X-C           0.000000000000000e+00     0.00000e+00    1
------------------------------------------------------------
 Electronic   -7.110131319967708e+00    -2.55576e-03    0
 Nuclear      -1.890205386758395e-01     6.08507e-07    0
------------------------------------------------------------
 Total        -7.299151858643548e+00    -2.55515e-03    0
============================================================


################ Wall time: 5.22556e+01 sec ################




####################### SCF cycle  4 #######################


============================================================
 Current precision                              1.00000e-05
------------------------------------------------------------
 Orbital threshold                              1.00000e-03
 Property threshold                             1.00000e-06
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
 Nuclear energy                      -1.890205103279445e-01
 Electronic energy                   -7.110358950921537e+00
------------------------------------------------------------
 Total energy                        -7.299379461249481e+00
------------------------------------------------------------
                 Wall time: 9.72471e-01 sec 
============================================================


============================================================
                Applying Helmholtz operators
------------------------------------------------------------
 Orb    RealNorm   Nodes     ImagNorm   Nodes     Timing
------------------------------------------------------------
  0    1.00055e+00   520    0.00000e+00     0   8.14550e+00
------------------------------------------------------------
                 Wall time: 8.14709e+00 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-05
------------------------------------------------------------
 Coulomb density                       584      1.48266e-01
 Coulomb potential                     328      3.33507e+00
4.000000000000000e+00
1.198458619540644e-03 1
1
2.109973967411145e-06 2
2
converged V_n
 Hartree-Fock exchange                 520      3.40911e+00
------------------------------------------------------------
                 Wall time: 4.56652e+01 sec 
============================================================


============================================================
                  Calculating Fock matrix
------------------------------------------------------------
 Kinetic part                         3.454232140000000e-01
 Potential part                       5.084272840000000e-01
------------------------------------------------------------
                 Wall time: 8.54014e-01 sec 
============================================================


============================================================
                          Orbitals
------------------------------------------------------------
 Orb    F(i,i)        Error         nNodes  Spin  Occ  Done 
------------------------------------------------------------
  0  -2.66595e+00   8.30162e-04        520    p    2    1
------------------------------------------------------------
 Total error:                            8.30162e-04    1
============================================================


============================================================
                    Energy                 Update      Done 
------------------------------------------------------------
 Orbital      -5.330872164475030e+00    -2.39961e-03    0
 Kinetic       7.511520965958107e+00    -3.46983e-02    0
 N-E          -1.614931771221109e+01     3.66426e-02    0
 Coulomb       3.306924500634326e+00    -4.34386e-03    0
 Exchange     -1.653462209745351e+00     2.17189e-03    0
 X-C           0.000000000000000e+00     0.00000e+00    1
------------------------------------------------------------
 Electronic   -7.110358950921537e+00    -2.27631e-04    0
 Nuclear      -1.890205103279445e-01     2.83479e-08    1
------------------------------------------------------------
 Total        -7.299379461249481e+00    -2.27603e-04    0
============================================================


################ Wall time: 5.57434e+01 sec ################




####################### SCF cycle  5 #######################


============================================================
 Current precision                              1.00000e-05
------------------------------------------------------------
 Orbital threshold                              1.00000e-03
 Property threshold                             1.00000e-06
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
 Nuclear energy                      -1.890205275120087e-01
 Electronic energy                   -7.110385292676900e+00
------------------------------------------------------------
 Total energy                        -7.299405820188909e+00
------------------------------------------------------------
                 Wall time: 9.50302e-01 sec 
============================================================


============================================================
                Applying Helmholtz operators
------------------------------------------------------------
 Orb    RealNorm   Nodes     ImagNorm   Nodes     Timing
------------------------------------------------------------
  0    1.00022e+00   520    0.00000e+00     0   8.46387e+00
------------------------------------------------------------
                 Wall time: 8.46497e+00 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-05
------------------------------------------------------------
 Coulomb density                       584      1.32755e-01
 Coulomb potential                     328      3.40772e+00
4.000000000000000e+00
5.162298193832275e-04 1
1
6.144101915183089e-07 2
2
converged V_n
 Hartree-Fock exchange                 520      3.43141e+00
------------------------------------------------------------
                 Wall time: 4.56980e+01 sec 
============================================================


============================================================
                  Calculating Fock matrix
------------------------------------------------------------
 Kinetic part                         2.864446620000000e-01
 Potential part                       4.543079290000001e-01
------------------------------------------------------------
                 Wall time: 7.40862e-01 sec 
============================================================


============================================================
                          Orbitals
------------------------------------------------------------
 Orb    F(i,i)        Error         nNodes  Spin  Occ  Done 
------------------------------------------------------------
  0  -2.66618e+00   2.69259e-04        520    p    2    1
------------------------------------------------------------
 Total error:                            2.69259e-04    1
============================================================


============================================================
                    Energy                 Update      Done 
------------------------------------------------------------
 Orbital      -5.331908732442307e+00    -1.03657e-03    0
 Kinetic       7.497666381532701e+00    -1.38546e-02    0
 N-E          -1.613447921901377e+01     1.48385e-02    0
 Coulomb       3.304904009761612e+00    -2.02049e-03    0
 Exchange     -1.652451957242233e+00     1.01025e-03    0
 X-C           0.000000000000000e+00     0.00000e+00    1
------------------------------------------------------------
 Electronic   -7.110385292676900e+00    -2.63418e-05    0
 Nuclear      -1.890205275120087e-01    -1.71841e-08    1
------------------------------------------------------------
 Total        -7.299405820188909e+00    -2.63589e-05    0
============================================================


################ Wall time: 5.59777e+01 sec ################




####################### SCF cycle  6 #######################


============================================================
 Current precision                              1.00000e-05
------------------------------------------------------------
 Orbital threshold                              1.00000e-03
 Property threshold                             1.00000e-06
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
 Nuclear energy                      -1.890205355262134e-01
 Electronic energy                   -7.110388819040398e+00
------------------------------------------------------------
 Total energy                        -7.299409354566611e+00
------------------------------------------------------------
                 Wall time: 8.77839e-01 sec 
============================================================


============================================================
                Applying Helmholtz operators
------------------------------------------------------------
 Orb    RealNorm   Nodes     ImagNorm   Nodes     Timing
------------------------------------------------------------
  0    1.00009e+00   520    0.00000e+00     0   8.13308e+00
------------------------------------------------------------
                 Wall time: 8.13391e+00 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-05
------------------------------------------------------------
 Coulomb density                       584      1.45933e-01
 Coulomb potential                     328      3.46589e+00
4.000000000000000e+00
2.192053457723874e-04 1
1
1.449731454815510e-07 2
2
converged V_n
 Hartree-Fock exchange                 520      3.39809e+00
------------------------------------------------------------
                 Wall time: 4.24208e+01 sec 
============================================================


============================================================
                  Calculating Fock matrix
------------------------------------------------------------
 Kinetic part                         2.660371480000000e-01
 Potential part                       4.745798610000000e-01
------------------------------------------------------------
                 Wall time: 7.40716e-01 sec 
============================================================


============================================================
                          Orbitals
------------------------------------------------------------
 Orb    F(i,i)        Error         nNodes  Spin  Occ  Done 
------------------------------------------------------------
  0  -2.66627e+00   9.95890e-05        520    p    2    1
------------------------------------------------------------
 Total error:                            9.95890e-05    1
============================================================


============================================================
                    Energy                 Update      Done 
------------------------------------------------------------
 Orbital      -5.332357713994751e+00    -4.48982e-04    0
 Kinetic       7.492120602329265e+00    -5.54578e-03    0
 N-E          -1.612849149952407e+01     5.98772e-03    0
 Coulomb       3.304013087057941e+00    -8.90923e-04    0
 Exchange     -1.652006495457916e+00     4.45462e-04    0
 X-C           0.000000000000000e+00     0.00000e+00    1
------------------------------------------------------------
 Electronic   -7.110388819040398e+00    -3.52636e-06    1
 Nuclear      -1.890205355262134e-01    -8.01420e-09    1
------------------------------------------------------------
 Total        -7.299409354566611e+00    -3.53438e-06    1
============================================================


################ Wall time: 5.22705e+01 sec ################


============================================================
                      Convergence rate
------------------------------------------------------------
Iter    OrbError       Property                   Update  
------------------------------------------------------------
  1    6.00009e-02    -7.253415855431685e+00   -7.25342e+00
  2    1.26234e-02    -7.296596710591541e+00   -4.31809e-02
  3    3.03136e-03    -7.299151858643548e+00   -2.55515e-03
  4    8.30162e-04    -7.299379461249481e+00   -2.27603e-04
  5    2.69259e-04    -7.299405820188909e+00   -2.63589e-05
  6    9.95890e-05    -7.299409354566611e+00   -3.53438e-06
------------------------------------------------------------
                      SCF converged!!!                      
============================================================


============================================================
                  Digonalizing Fock matrix
------------------------------------------------------------
 Computing Lowdin matrix                        2.26395e-03
 Diagonalizing matrix                           2.20050e-02
 Rotating orbitals                              3.78580e-02
------------------------------------------------------------
                 Wall time: 6.22321e-02 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-05
------------------------------------------------------------
 Coulomb density                       584      1.38563e-01
 Coulomb potential                     328      3.24184e+00
4.000000000000000e+00
4.727442554519314e-08 1
1
converged V_n
 Hartree-Fock exchange                 520      3.40909e+00
------------------------------------------------------------
                 Wall time: 4.05693e+01 sec 
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
------------------------------------------------------------
                 Wall time: 8.31962e-01 sec 
============================================================


============================================================
                        Fock matrix
------------------------------------------------------------
-2.66627e+00
============================================================


============================================================
                      Orbital energies
------------------------------------------------------------
    n  spin  occ                            epsilon  
------------------------------------------------------------
    0    p    2                      -2.666273539076468e+00
============================================================


============================================================
                          Molecule
------------------------------------------------------------
 Nr  Element             x             y             z      
------------------------------------------------------------
  1     Li          0.00000e+00   0.00000e+00   0.00000e+00
------------------------------------------------------------
 Center of mass:    0.00000e+00   0.00000e+00   0.00000e+00
============================================================


                                                            
============================================================
                         SCF Energy                         
------------------------------------------------------------
                                                            
 Sum orbital energy:                 -5.332547078152936e+00
 Kinetic energy:                      7.489911914215202e+00
 E-N energy:                         -1.612609445361047e+01
 Coulomb energy:                      3.303635365233254e+00
 Exchange energy:                    -1.651817634612089e+00
 X-C energy:                          0.000000000000000e+00
 El. External field energy:           0.000000000000000e+00
 Nuc. External field energy:          0.000000000000000e+00
 Reac. field energy:                 -6.299602300738866e-02
 Reac. field el. energy:              1.260245157249255e-01
 Reac. field nuc. energy:            -1.890205387323143e-01
                                                            
------------------------------------------------------------
                                                            
 Electronic energy:                  -7.110389324499026e+00
 Nuclear energy:                     -1.890205387323143e-01
                                                            
------------------------------------------------------------
                                                            
 Total energy       (au)             -7.299409863231340e+00
                    (kJ/mol)         -1.916459782213814e+04
                    (kcal/mol)       -4.580448807289661e+03
                    (eV)             -1.986270524261558e+02
                                                            
============================================================
                                                            



************************************************************
***                                                      ***
***                    Exiting MRChem                    ***
***                                                      ***
***               Wall time: 6.793069e+02                ***
***                                                      ***
************************************************************


